packages/compartment-mapper/src/parse-cjs-shared-export-wrapper.js

```diff
@@ -131,7 +140,20 @@ export const wrap = (moduleEnvironmentRecord, compartment, resolvedImports) => {
     } else {
       return namespace;
     }
-  });
+  };
+  if (requireResolve) {
+    require.resolve = freeze((specifier, options) =>
+      requireResolve(location, specifier, options),
+    );
+  } else {
+    require.resolve = freeze(specifier => {
+      const error = Error(`Cannot find module '${specifier}'`);
+      defineProperty(error, 'code', { value: 'MODULE_NOT_FOUND' });
+      throw error;
```

Comment:
Question: Should this be indistinguishable from an error Node would throw, or rather match it loosely but leave a hint somewhere to it resulting from a lack of requireResolve option being passed to Endo?

Reply:
from a meeting: add the second line to the error mesage explaining requireResolve needs to be passed